packages/types/src/Metadata/v2/Calls.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import Struct from '../../codec/Struct';
+import Vector from '../../codec/Vector';
+import Text from '../../Text';
+import { MetadataName } from './MetadataRegistry';
+
+export class MetadataCallArg extends Struct {
+  constructor (value?: any) {
+    super({
+      name: Text,
+      type: MetadataName
+    }, value);
+  }
+
+  /**
+   * @description The argument name
+   */
+  get name (): Text {
+    return this.get('name') as Text;
+  }
+
+  /**
+   * @description The [[MetadataName]]
+   */
+  get type (): MetadataName {
+    return this.get('type') as MetadataName;
+  }
+}
+
+/**
+ * @name MetadataCall
+ * @description
+ * The definition of a call
+ */
+export class MetadataCall extends Struct {
+  constructor (value?: any) {
+    super({
+      // NOTE Just commented out, it is not available in current, but an open question
```

Comment:
You could remove this now, should actually do so in V1 as well - it is always a chicken-and-egg while these things are ongoing, you first want the JS done before the substrate merge so the whole world doesn't break, so left the comment and never came back to just clean up.